internal/cluster/service.go

```diff
@@ -20,11 +18,7 @@ import (
 )
 
 const (
-	// ClusterSubstrAllowedRegexp :TEMPFIX: :chmouel:
-	// filter cluster url on name with a regexp until we have a proper way to do
-	// this see
-	// https://chat.openshift.io/developers/pl/j6ccecpebfr1zj4xxfu4cppm9y
-	ClusterSubstrAllowedRegexp = "(start|free)"
+	osioType = "OSO"
```

Comment:
Are there any chances cluster service could change it?
Its going to use for filter out [OSO clusters](https://github.com/fabric8-services/fabric8-jenkins-idler/pull/329/files#diff-47673f961fb8ff14f936300ee7dd4988R95).

cc: @dipak-pawar